### PR TITLE
[VEN-1996]: add deployments of agEUR market in Stablecoins pool

### PR DESCRIPTION
**Pull request by GitGuru7**
Resolves VEN-1996

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -1423,6 +1454,23 @@ export const globalConfig: NetworkConfig = {
...
+            symbol: "vagEUR_Stablecoins",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: convertToUnit("0.02", 18),
+            multiplierPerYear: convertToUnit("0.1", 18),
+            jumpMultiplierPerYear: convertToUnit("2.5", 18),
+            kink_: convertToUnit("0.5", 18),
+            collateralFactor: convertToUnit("0.75", 18),
+            liquidationThreshold: convertToUnit("0.8", 18),
+            reserveFactor: convertToUnit("0.1", 18),
+            initialSupply: convertToUnit(10000, 18),
```
Comment (chechu) [inline helpers/deploymentConfig.ts:1469]:
```suggestion
            initialSupply: convertToUnit(9000, 18),
```

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -1423,6 +1454,23 @@ export const globalConfig: NetworkConfig = {
...
+            supplyCap: convertToUnit(100000, 18),
+            borrowCap: convertToUnit(50000, 18),
+            vTokenReceiver: "0x3DdfA8eC3052539b6C9549F12cEA2C295cfF5296",
```
Comment (chechu) [inline helpers/deploymentConfig.ts:1472]:
```suggestion
            vTokenReceiver: "0xc444949e0054a23c44fc45789738bdf64aed2391",
```

File: helpers/deploymentConfig.ts (hunk 1 above)
Comment (GitGuru7) [reply helpers/deploymentConfig.ts]: [e9eb707](https://github.com/VenusProtocol/isolated-pools/pull/309/commits/e9eb707e871cea81a6c2c5be57510ec4fb5c49d8)

File: helpers/deploymentConfig.ts (hunk 2 above)
Comment (GitGuru7) [reply helpers/deploymentConfig.ts]: [e9eb707](https://github.com/VenusProtocol/isolated-pools/pull/309/commits/e9eb707e871cea81a6c2c5be57510ec4fb5c49d8)

**Comment by chechu** [comment]
After executing https://app.venus.io/#/governance/proposal/170 the Shortfall and the ProtocolShareReserve contracts in the VToken should be set to the following values:

Mainnet:
* Shortfall: 0xf37530A8a810Fcb501AA0Ecd0B0699388F0F2209
* ProtocolShareReserve: 0xfB5bE09a1FA6CFDA075aB1E69FE83ce8324682e4

Testnet:
* Shortfall: 0x503574a82fE2A9f968d355C8AAc1Ba0481859369
* ProtocolShareReserve: 0xc987a03ab6C2A5891Fc0919f021cc693B5E55278

I would change the [deployment script of the VToken contracts](https://github.com/VenusProtocol/isolated-pools/blob/develop/deploy/009-deploy-vtokens.ts#L117) to consider this change, and I would redeploy the vagEUR contract

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**